processing/src/test/java/io/druid/query/aggregation/AggregationTestHelper.java

```diff
@@ -196,10 +208,22 @@ public static final AggregationTestHelper createSelectQueryAggregationTestHelper
         QueryRunnerTestHelper.NOOP_QUERYWATCHER
     );
 
+    IndexIO indexIO = new IndexIO(
```

Comment:
It depends on specific ObjectMapper which gets appropriate Modules from Aggregator impl registered as provided by the user of this class. that is the reason it is not reused.